small additions (#39)
movestore/movestore.github.io: programing_move2.md

-   `mt_time_column()`: returns the name of the column containing the timestamps used by the `move2` object
-   `mt_track_id_column()`: returns the name of the column containing the track ids used by the `move2` object

> Note that locations can most generically be addressed using the `sf` package, as the `move2` objects are of this class.

Reviewer: ```suggestion

```

> -	`st_coordinates()`: returns locations of each even in the track for type `move2_loc`.

Reviewer: ```suggestion
- `sf::st_coordinates()`: returns the coordinates of the events from the track(s) of a `move2` object
```
